.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDateTimePeriodParser.cs

```diff
@@ -397,6 +402,10 @@ protected virtual DateTimeResolutionResult ParseSpecificTimeOfDay(string text, D
                     if (!valid)
                     {
                         ers = this.Config.DateExtractor.Extract(afterStr, referenceTime);
+                        if ((Config.Options & DateTimeOptions.TasksMode) != 0)
```

Comment:
Same, add a blank line. These help in review/merge and follow the workflow (as in the original code).

Reply:
added blank line.